### PR TITLE
Modified postgres folder path for clarity

### PR DESCRIPTION
**Pull request by coltoneshaw**
#### Summary
We specify an absolute path of `/etc/postgresql/10/main` when Ubuntu 20.04 installs version 12. Replaced the path and added a small amount of instructions to make this more clear.

### CODE REVIEW

File: source/install/install-ubuntu-2004-postgresql.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ Assume that the IP address of this server is 10.10.10.1.
 
   ``exit``
 
-9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/10/main/postgresql.conf`` in a text editor as *root* user. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
+9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/{version}/main/postgresql.conf`` in a text editor as *root* user. You will need to replace `{version}` with the version PostgreSQL is currently running. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```
Comment (cwarnermm) [inline source/install/install-ubuntu-2004-postgresql.rst:50]:
```suggestion
9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/{version}/main/postgresql.conf`` in a text editor as *root* user. Replace `{version}` with the version of PostgreSQL that is currently running. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

Comment (cwarnermm) [reply source/install/install-ubuntu-2004-postgresql.rst:50]: Thank you! We use the same hard-coded convention for the Ubuntu 18.04 installation documentation as well (https://docs.mattermost.com/install/install-ubuntu-1804.html#) Could you make the same change there for consistency, please?

**Review by cwarnermm — CHANGES_REQUESTED** [review]
Thank you for adding clarity to the product documentation. This update affects an additional installation page for Ubuntu 18.04.

**Comment by cwarnermm** [comment]
@justinegeffen - Would you be able to make this change to a second file as well, please? This change would be beneficial on the installation page for Ubuntu 18.04.

File: source/install/install-ubuntu-2004-postgresql.rst [hunk 2]
```diff
@@ -47,7 +47,7 @@ Assume that the IP address of this server is 10.10.10.1.
 
   ``exit``
 
-9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/10/main/postgresql.conf`` in a text editor as *root* user. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
+9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/{version}/main/postgresql.conf`` in a text editor as *root* user. Replace `{version}` with the version of PostgreSQL that is currently running. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:50]:
```suggestion
9. (Optional) If you use a different server for your database and the Mattermost server, you may allow PostgreSQL to listen on all assigned IP addresses. To do so, open ``/etc/postgresql/{version}/main/postgresql.conf`` in a text editor as *root* user. Replace ``{version}`` with the version of PostgreSQL that's currently running. As a best practice, ensure that only the Mattermost server is able to connect to the PostgreSQL port using a firewall.
```

File: source/install/install-ubuntu-2004-postgresql.rst [hunk 3]
```diff
@@ -65,7 +65,7 @@ Assume that the IP address of this server is 10.10.10.1.
 
   **If the Mattermost server and the database are on the same machine**:
```
Comment (justinegeffen) [inline source/install/install-ubuntu-2004-postgresql.rst:66]:
```suggestion
  **If the Mattermost server and the database are on the same machine:**
```

**Comment by justinegeffen** [comment]
> @justinegeffen - Would you be able to make this change to a second file as well, please? This change would be beneficial on the installation page for Ubuntu 18.04.
Done. :)

**Review by cwarnermm — APPROVED** [review]
Thank you so much for applying these changes!

File: source/install/install-ubuntu-1804-postgresql.rst [hunk 4]
```diff
@@ -100,9 +101,10 @@ Assume that the IP address of this server is 10.10.10.1.
 
     ``psql --host={postgres-server-IP} --dbname=mattermost --username=mmuser --password``
 
-    .. note::
-      You might have to install the PostgreSQL client software to use the command.
+.. note::
```
Comment (amyblais) [inline source/install/install-ubuntu-1804-postgresql.rst:104]: @justinegeffen The last "note" section (same for the second file) is not properly formatted (minor issue).

**Pull request merged**